homeassistant/components/peco/sensor.py

```diff
@@ -0,0 +1,116 @@
+"""Sensor component for PECO outage counter."""
+import asyncio
+from datetime import timedelta
+from types import MappingProxyType
+from typing import Any, Final, cast
+
+from peco import BadJSONError, HttpError
+
+from homeassistant.components.sensor import (
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+    UpdateFailed,
+)
+
+from .const import DOMAIN, LOGGER, SCAN_INTERVAL
+
+PARALLEL_UPDATES: Final = 0
+SENSOR_LIST = (
+    SensorEntityDescription(key="customers_out", name="Customers Out"),
+    SensorEntityDescription(
+        key="percent_customers_out",
+        name="Percent Customers Out",
+        native_unit_of_measurement=PERCENTAGE,
+    ),
+    SensorEntityDescription(key="outage_count", name="Outage Count"),
+    SensorEntityDescription(key="customers_served", name="Customers Served"),
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the sensor platform."""
+    api = hass.data[DOMAIN][config_entry.entry_id]
+    websession = async_get_clientsession(hass)
+    conf: MappingProxyType[str, Any] = config_entry.data
```

Comment:
since "county" is the only key in this dict it could be extracted right here and used directly afterwards
```suggestion
    county: str = config_entry.data["county"]
```